Bindings/Java/Matlab/Utilities/osimC3Dconverter.m

```diff
@@ -0,0 +1,258 @@
+function osimC3Dconverter(varargin)
+% Utility function for converting c3d data to OpenSim format.
+% Inputs to function are pairs of string-string or string-number pairs. 
+% 'filepath', 'path2file'   reads c3d data at path2file.
+% 'axis', 'X'               applies first rotation to axis X (string)
+% 'value', 90               value, in degrees, to rotate data by
+%
```

Comment:
Missed this last time, but what are the Outputs of the converter? A markers.trc and a forces.mot file?